### PR TITLE
Increases clarity in staging process.

### PR DESCRIPTION
**Pull request by chicken-biryani**
PR Checklist:

- [X] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [X] put any other relevant information below

This PR hopes to increase clarity for a reader by:
- Adding the steps that are there in the staging process. So that the person hopefully doesn't skip them.
- Adding paths so that it is simplified for the reader.
- Modifying the words a little.

### CODE REVIEW

**Comment by chicken-biryani** [comment]
Hi,
This PR hopes to increase clarity for the reader.

**A thing that I have observed is that:**

As a reader for 'getting started' I kept _jumping from link to link without clearly reading all the instructions._

For example: the sentence - Look at [the example recipe](https://github.com/conda-forge/staged-recipes/tree/master/recipes/example) in the [staged-recipes repository](https://github.com/conda-forge/staged-recipes)`
That's the first instruction I read. I clicked on the link and I thought to myself-- "Okay. Now what?"

And I am assuming there are many readers like me (more experienced ofc!) who fall into a pile of mishaps they could have avoided if only they had read all the instructions.

**A possible solution**:

Shall we put up the instructions on going around Grayskull, CRAN (Not my forte!), and the example recipe within this document itself?
To neaten things up we could make use of [sphinx tabs.](https://sphinx-tabs.readthedocs.io/en/latest/)

We can also include the blo

**Comment by chicken-biryani** [comment]
@kathatherine @beckermr @BastianZim do you think it's a good idea to do so?

**Comment by kathatherine** [comment]
@chicken-biryani I love sphinx tabs. Anaconda, where I work, is thinking of adding that to our own docs. Please write up an example of what you're thinking and I'd love to look at it. Anything that can increase clarity for the user and keep them from wandering away from the documentation flow is a good idea to me.

**Review by kathatherine — CHANGES_REQUESTED** [review]

**Comment by chicken-biryani** [comment]
@kathatherine 
Fixed it.
(i.e. `...`) does seem to be better.

>  Please write up an example of what you're thinking
I'll create a separate draft PR for that? It may get a little messy but I think it's worth a shot.

File: src/maintainer/adding_pkgs.rst [hunk 1]
```diff
@@ -49,17 +54,18 @@ Step-by-step Instructions
...
-#. Create a new branch from the staged-recipes ``master`` branch.
-#. Within your forked copy, generate a new folder in the recipes subdirectory
-   and copy the `meta.yml
+#. Fork and clone the `staged-recipes
+   <https://github.com/conda-forge/staged-recipes>`_
+   repository from GitHub.
+#. Checkout a new branch from the staged-recipes ``main`` branch.
+#. Through CLI, cd inside the 'staged-recipes/recipes' directory.
+#. Within your forked copy, create a new folder in the recipes folder for your package (i.e, ``...staged-recipes/recipes/<name-of-package>``)
+#. Copy `meta.yml
```
Comment (jaimergp) [inline src/maintainer/adding_pkgs.rst:63]: I think this is a typo, right?

```suggestion
#. Copy `meta.yaml
```

Comment (chicken-biryani) [reply src/maintainer/adding_pkgs.rst:63]: Ah! Yes. Thanks for catching that.

Comment (jaimergp) [reply src/maintainer/adding_pkgs.rst:63]: Btw, what I usually do in this step is:

```console
# unix
cp -r recipes/example recipes/<package_name>
# windows
xcopy /h /e /i "recipes\example\" "recipes\<package_name>"
```

, so this step and the previous are the same one. But it's essentially the same, consider it a tip!

**Comment by kathatherine** [comment]
@chicken-biryani Yes, a separate PR would probably be the way to go. Maybe even an entirely separate issue?

**Comment by chicken-biryani** [comment]
@kathatherine on it!

**Review by kathatherine — APPROVED** [review]
This looks good to me!

**Comment by jakirkham** [comment]
Powercycling for CI

**Review by jakirkham — APPROVED** [review]

**Comment by chicken-biryani** [comment]
Hi,

I am clubbing my ideas for the adding packages into a single PR over here:

https://github.com/conda-forge/conda-forge.github.io/pull/1755

Will wind up the work asap!

**Pull request merged**